### PR TITLE
Fixes #249 - Time accounting enhancements - admin documentation.

### LINKED ISSUE #249: Time accounting enhancements - admin documentation
* Time Accounting
  * Units
  * Activity Types
  * Split of the management screen into separate tabs

### PR DESCRIPTION
**Pull request by dvuckovic**

### CODE REVIEW

File: manage/time-accounting.rst [hunk 1]
```diff
@@ -2,30 +2,34 @@ Time Accounting
...
-Zammad's time accounting uses ticket selectors (filters) to check if a ticket
-is applicable for time accounting or not. If a ticket applies for accounting,
-Zammad will request the agent to provide how much time was needed to process
-the current ticket step.
+Settings
+~~~~~~~~
+
+Zammad's time accounting uses ticket **Selector** (filter) to check if a ticket
+is considered for the time accounting or not. If a ticket is applicable, Zammad
+will request the agent to provide how much time was needed to process the
```
Comment (fliebe92) [inline manage/time-accounting.rst:19]: `many` instead of `much`? 🤔

File: manage/time-accounting.rst [hunk 2]
```diff
@@ -2,30 +2,34 @@ Time Accounting
...
 
-      However, the time accounting dialogue is not mandatory and can be
-      cancelled by your agents if needed. You cannot enforce time accounting.
+      However, the time accounting dialog is not mandatory and can be canceled
+      by your agents if needed. You cannot enforce time accounting.
 
-      If a ticket no longer applies for time accounting, the already accounted
-      time is not lost.
+      If a ticket is no longer considered for the time accounting, the already
+      accounted time will be.
```
Comment (fliebe92) [inline manage/time-accounting.rst:32]: Incomplete sentence. 🤔

File: manage/time-accounting.rst (hunk 1 above)
Comment (dvuckovic) [reply manage/time-accounting.rst:19]: I disagree: the `time` noun is not countable, so it's always considered as singular. Therefore, we describe it with `much`, while `many` is reserved to countable nouns in plural form (e.g. `items`, `tasks`, etc).

File: manage/time-accounting.rst (hunk 2 above)
Comment (dvuckovic) [reply manage/time-accounting.rst:32]: Oops, fixed!

File: manage/time-accounting.rst (hunk 1 above)
Comment (fliebe92) [reply manage/time-accounting.rst:19]: Got it, thanks.

**Review by fliebe92 — APPROVED** [review]

**Pull request merged**